integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/classMediator/PropertyPersistenceDeletingAndAddingTstCase.java

```diff
@@ -37,39 +42,46 @@
     private static final String CLASS_JAR_FIVE_PROPERTIES="org.wso2.carbon.test.mediator.stockmediator-v1.0.jar";
     private static final String CLASS_JAR_FOUR_PROPERTIES="org.wso2.carbon.test.mediator.stockmediator-v1.0.2.jar";
     private static final String JAR_LOCATION= "/artifacts/ESB/jar";
+    private HashMap<String, String> startupParameterMap = new HashMap<String, String>();
+    private CarbonTestServerManager server;
 
+    private static final String osname = System.getProperty("os.name").toLowerCase().toString();
     private ServerConfigurationManager serverConfigurationManager;
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
-
-        super.init();
-        serverConfigurationManager=new ServerConfigurationManager(context);
-        serverConfigurationManager.copyToComponentLib
-                (new File(getClass().getResource(JAR_LOCATION + File.separator + CLASS_JAR_FIVE_PROPERTIES).toURI()));
-        serverConfigurationManager.restartGracefully();
-
-        super.init();
-        loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/class/class_property_persistence_five_properties.xml");
+        if (!(System.getProperty("os.name").toLowerCase().contains("windows"))) {
```

Comment:
We can use the osname variable instead of System.getProperty("os.name").toLowerCase().

Reply:
Fixed